tests/snmpsim/linux_raspberrypi.snmprec

```diff
@@ -1,11 +1,11 @@
-1.3.6.1.2.1.1.1.0|4|Linux rpi 4.14.98+ #1200 Tue Feb 12 20:11:02 GMT 2019 armv6l
+1.3.6.1.2.1.1.1.0|4|Linux mso 4.19.97+ #1294 Thu Jan 30 13:10:54 GMT 2020 armv6l
```

Comment:
It doesn't look like there is any significant change in the snmp data here, just the values are changing.  It would be best to leave the original snmprec in this case so that it is easier to see what is changed by your PR.

Reply:
There is also an extra line with `WMV9=disabled` below. I will fix the extend script in another repo.